twilio/table_twilio_account_bundles.go

```diff
@@ -0,0 +1,122 @@
+package twilio
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+	openapi "github.com/twilio/twilio-go/rest/numbers/v2"
+)
+
+func tableTwilioAccountBundles(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "twilio_account_bundles",
```

Comment:
Please update the table name as `twilio_account_bundle`, we maintain our table names in singular format. 
Also please update the file names as well.